.github/workflows/sonarqube.yml

```
@@ -0,0 +1,24 @@
+name: SonarQube Checks
+on:
+  # Trigger analysis when pushing to your main branches, and when creating a pull request.
+  push:
+    branches:
+      - main
+      - master
+      - develop
+      - 'release/**'
```

for the release candidates
```suggestion
      - 'release/**'
      - 'rc/**'
```